src/test/java/nl/tudelft/ewi/devhub/server/database/controllers/WarningTest.java

```diff
@@ -0,0 +1,94 @@
+package nl.tudelft.ewi.devhub.server.database.controllers;
+
+import nl.tudelft.ewi.devhub.server.database.entities.Delivery;
+import nl.tudelft.ewi.devhub.server.database.entities.Group;
+import nl.tudelft.ewi.devhub.server.database.entities.PullRequest;
+import nl.tudelft.ewi.devhub.server.web.models.PullRequestWarningResponse;
+import nl.tudelft.ewi.devhub.server.web.resources.WarningResource;
+import nl.tudelft.ewi.git.client.Repository;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Matchers.any;
+
+
+/**
+ * Created by LC on 02/05/15.
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class WarningTest {
+        @Mock
+        private Deliveries deliveries;
+
+        @Mock
+        private PullRequests pullRequest;
+
+        @Mock
+        private Group group;
+
+        @Mock
+        private Repository repository;
+
+        @Mock
+        private PullRequest mergedRequest;
+
+        @Mock
+        private Delivery delivery;
+
+
+        private WarningResource warningResource;
+
+        @Before
+        public void setup() {
+            warningResource = new WarningResource(group, null, pullRequest, deliveries, null, repository);
```

Comment:
I recently found out the [`@InjectMocks`](http://docs.mockito.googlecode.com/hg/1.9.5/org/mockito/InjectMocks.html) annotation which is perfect for this case.